index.html

```diff
@@ -10,7 +10,7 @@
 <div class="main-explain-area jumbotron">
   <h2><center>To be held as part of
   <a href='https://www.escience-conference.org/2022/'>eScience 2022</a>,
-  Salt Lake City, Utah, October 11-14, 2022.
+  Salt Lake City, Utah, October 10-14, 2022.
```

Comment:
Maybe we should change the date to just the data of the workshop, not the date of the whole conference